src/main/java/codeu/model/data/User.java

```diff
@@ -56,6 +60,8 @@ public User(UUID id, String name, String passwordHash, Instant creation, boolean
     this.passwordHash = passwordHash;
     this.creation = creation;
     this.isAdmin = isAdmin;
+
+    this.conversationSet = new HashSet<>();
```

Comment:
Add a parameter for conversationSet

Reply:
I initially did that, but then realized I would have to refactor every single user creation instance.

A conversation set can be set with `setConversationSet` (starting to feel like Dr. Seuss with all this assonance)